packages/database/env.example

```diff
@@ -0,0 +1,4 @@
+NODE_ENV=development
+
+DATABASE_URL=postgres://latitude:secret@localhost:5432/latitude_development
+TEST_DATABASE_URL=postgres://latitude:secret@localhost:5432/latitude_test
```

Comment:
I will find a way to share .env files between packages in the monorepo in another pr

Reply:
with another package xd